### ISSUE TITLE
"No executable found" when running Stream Analytics Job Locally

### ISSUE THREAD

**Issue by davefrancis** [microsoft/vscode-asa#13]
When running a Stream Analytics job locally with the extension, the job fails with the following error:
```
[Info] Checking .NET Core environment ...
[Info] .NET Core environment is ready.
[Info] Local Run Server is ready.
[Info] Started Local Run: C:\Users\David Richards\AppData\Roaming\ASALocalRunService\0.2\LocalRunService.dll
[Error] No executable found matching command "dotnet-C:\Users\David"
```

I believe the issue may occur because my user folder has a space in it as I get the same error when I run: `dotnet C:\Users\David Richards\AppData\Roaming\ASALocalRunService\0.2\LocalRunService.dll` in powershell. When I wrap the path in quotes the executable runs properly.

I'm using windows 10 with ms-bigdatatools.vscode-asa version 0.1.3

**Comment by ms-guizha** [comment]
thanks for your feedback, we will fix it as soon as possible

**Comment by ms-guizha** [comment]
we had released the new bits with this fix

**Comment by su-jie** [comment]
The fix has been included in version 0.1.4.

**Comment by su-jie** [comment]
This issue has been fixed in 0.1.4.
